homeassistant/components/media_player/kodi.py

```diff
@@ -301,4 +301,4 @@ def media_seek(self, position):
 
     def play_media(self, media_type, media_id, **kwargs):
         """Send the play_media command to the media player."""
-        self._server.Player.Open({media_type: media_id}, {})
+        self._server.Player.Open({"item": {"file": str(media_id)}})
```

Comment:
You are now hardcoding media_type to be `file` instead of allowing the user to specify it. Does Kodi have any other media types?